doc/source/release/2.0.0-notes.rst

```diff
@@ -1,45 +1,313 @@
-.. currentmodule:: numpy
-
-==========================
+=========================
 NumPy 2.0.0 Release Notes
-==========================
-
-
-Highlights
-==========
+=========================
 
 
 New functions
 =============
 
+Matrix transpose support for ndarrays
+-------------------------------------
+NumPy now offers support for calculating the matrix transpose of an array. The
+matrix transpose is equivalent to swapping the last two axes of an array. Both
+``np.ndarray`` and ``np.ma.MaskedArray`` now expose an ``.mT`` attribute.
+
+(`gh-23762 <https://github.com/numpy/numpy/pull/23762>`__)
+
+
+NumPy 2.0 Python API removals
+=============================
+
+* ``np.geterrobj``, ``np.seterrobj`` and the related ufunc keyword argument
+  ``extobj=`` have been removed.  The preferred replacement for all of these
+  is using the context manager ``with np.errstate():``.
+
+  (`gh-23922 <https://github.com/numpy/numpy/pull/23922>`__)
+
+* ``np.cast`` has been removed. The literal replacement for
+  ``np.cast[dtype](arg)`` is ``np.asarray(arg, dtype=dtype)``.
+
+* ``np.source`` has been removed. The preferred replacement is 
+  ``inspect.getsource``.
+
+* ``np.lookfor`` has been removed.
+
+  (`gh-24144 <https://github.com/numpy/numpy/pull/24144>`__)
+
+* ``numpy.who`` has been removed. As an alternative for the removed functionality, one
+can use a variable explorer that is available in IDEs such as Spyder or Jupyter Notebook.
```

Comment:
Indentation is off.

Reply:
This part came generated from `towncrier` - Fixed it!

Reply:
It probably was in the fragment. That is not unusual :)